src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -2,8 +2,10 @@ import { Request, Response } from "express";
 import { envVars } from "config/env";
 import { EnvironmentEnum } from "interfaces/common";
 import { compact, map } from "lodash";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
 
 const instance = envVars.INSTANCE_NAME;
+const jiraHost = `https://${instance}.atlassian.net`;
```

Comment:
instance name is the jira app instance name, not the jira host name.

Reply:
gahhhhhh